sparkpost/transmissions.py

```diff
@@ -83,6 +86,8 @@ def send(self, **kwargs):
         :param dict substitution_data: Corresponds to substitutions in
             html/text content. See `substitutions reference
             <https://www.sparkpost.com/docs/substitutions-reference>`_.
+        :param str start_time: Delay generation of messages until this
```

Comment:
can we explicitly mention that datetime must come with tz? May be an example too.